FirebaseRemoteConfig.podspec

```diff
@@ -28,7 +28,6 @@ app update.
 
   base_dir = "FirebaseRemoteConfig/Sources/"
   s.source_files = base_dir + '**/*.[mh]'
-  s.requires_arc = base_dir + '*.m'
```

Comment:
This flag indicates that the other files should use arc.  Removing the flag indicates all files follow the default of building with arc.